### PR TITLE
Lean installation by default

### PR DESCRIPTION
**Pull request by tjs-intel**
Orchestrate (ie. `sigopt cluster` cli commands) can be installed via `pip install 'sigopt[orchestrate]'`. Documentation should be updated as appropriate.

### CODE REVIEW

File: sigopt/orchestrate/kubectl/service.py [hunk 1]
```diff
@@ -24,7 +25,8 @@ def __str__(self):
     )
 
 class KubectlService(Service):
-  kubectl_command = 'kubectl'
+  def kubectl_command(self):
```
Comment (dwanderson-intel) [inline sigopt/orchestrate/kubectl/service.py:28]: should it be `@property`?

**Review by dwanderson-intel — APPROVED** [review]
non-blocking comment; looks great!

File: sigopt/orchestrate/kubectl/service.py (hunk 1 above)
Comment (tjs-intel) [reply sigopt/orchestrate/kubectl/service.py:28]: The others aren't. So I could make them properties too or just change to getters

File: requirements.txt [hunk 2]
```diff
@@ -1,15 +1,8 @@
...
-boto3>=1.16.34,<2.0.0
 click>=8.0.0
-docker>=4.4.0,<5.0.0
-ipython>=5.0.0
-kubernetes>=12.0.1,<13.0.0
 packaging>=21.3
-pyOpenSSL>=20.0.0
 pypng>=0.0.20
 requests>=2.25.0,<3.0.0
-rsa>=4.7,<5.0.0
```
Comment (tjs-intel) [inline requirements.txt:14]: @dwanderson-intel any idea why this was in here?

Comment (dwanderson-intel) [reply requirements.txt:14]: dunno, looks like it came over in the large, 8.0.0 runs release. Could've been bundled as part of cluster management?

**Pull request merged**